deployment_scripts/deploy_linux.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 source config
 
+exists() {
+  command -v "$1" >/dev/null 2>&1
+}
+
```

Comment:
Unused here?

Reply:
Yeah, it's prep for later code. I can remove from this PR but it'll just sneak back in later on

Reply:
Then leave it, but I'm giving myself a gold star for effort 🌟